data/data-pipeline/data_pipeline/content/config/csv.yml

```diff
@@ -59,9 +59,15 @@ fields:
 - score_name: Definition N (communities) (based on adjacency index and low income alone)
   label: Identified as disadvantaged based on neighbors and relaxed low income threshold only
   format: bool
+- score_name: Identified as disadvantaged due to tribal overlap
+  label: Identified as disadvantaged due to tribal overlap
+  format: bool
 - score_name: Definition N community, including adjacency index tracts
   label: Identified as disadvantaged
   format: bool
+- score_name: Percentage of tract that is disadvantaged
```

Comment:
can we make clear this is area-based?

Reply:
@emma-nechamkin I asked Beth in the ticket; it's good by me, but I leave the actual words to Beth and others